utils_cv/tracking/model.py

```diff
@@ -243,9 +279,48 @@ def save(self, path) -> None:
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
 
-    def evaluate(self, results, gt) -> pd.DataFrame:
-        pass
+    def evaluate(self,
+                 results: Dict[int, List[TrackingBbox]],
+                 gt_root_path: str) -> pd.DataFrame:
```

Comment:
Don't think this ground truth root path is needed if we use tempfile.TemporaryDirectory(). See below.

Reply:
It's still used by ` evaluator = Evaluator(gt_root_path, "single_vid", "mot")` to locate the gt annotation file. I'll use tempfile.TemporaryDirectory() to write the transient `results.txt` file, before `results.txt` file was output into this groundtruth root path too.